airbyte-cron/src/main/resources/application.yml

```diff
@@ -0,0 +1,44 @@
+micronaut:
+  application:
+    name: airbyte-cron
+
+airbyte:
+  cron:
+    update-definitions:
+      enabled: ${UPDATE_DEFINITIONS_CRON_ENABLED:false}
```

Comment:
Curious, why default to false?

Reply:
Just because we don't want to enable this in OSS yet - more info in my other comment